apps/site/app/globals.css

```
@@ -0,0 +1,166 @@
+@import url("https://fonts.googleapis.com/css2?family=Syne:wght@700;800&family=JetBrains+Mono:wght@300;400;500&family=Inter:wght@300;400&display=swap");
```

Stylelint (17.3.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Syne:wght@700;800&family=JetBrains+Mono:wght@300;400;500&family=Inter:wght@300;400&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Syne:wght@700;800&family=JetBrains+Mono:wght@300;400;500&family=Inter:wght@300;400&display=swap"" (import-notation)

(import-notation)